### PR TITLE
Add IDE settings and update README.md

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Add common IDE code style and import definitions and update `README.md` with installation instructions.

### CODE REVIEW

**Comment by dbmalkovsky** [comment]
The failure is caused by a dependency issue as `OH-core:jar:1.9.jar` is not found:

`
[ERROR] Failed to execute goal on project openhospital-api: Could not resolve dependencies for project org.isf:openhospital-api:jar:0.0.1: Could not find artifact org.isf:OH-core:jar:1.9 in google-maven-central (https://maven-central.storage-download.googleapis.com/maven2/) -> [Help 1]
`

File: README.md [hunk 1]
```diff
@@ -32,6 +32,40 @@ For the moment, to build this project you should
...
+* Select *Settings* in the *File* menu
+* Select *Editor*
+* Select *Code Style*
+* Expand the menu item and select *Java*
+* Go to *Scheme* at the top, click on the setting button by the side of the drop-down list
+* Select *Import Scheme*
+* Select *IntelliJ IDE code style XML*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/idea/OpenHospital-code-style-configuration.xml`
+* Select *OK* 
+* At this point the code style is stored as part of the IDE and is used for **all** projects opened in the editor.  To restrict the settings to just this project again select the setting button by the side of the *Scheme* list and select *Copy to Project...*. If successful a notice appears in the window that reads: *For current project*.
```
Comment (mwithi) [inline README.md:51]: I would suggest a collapsable section for the list, like this:

<details>
  <summary>IntelliJ instructions</summary>

* list1
* list2

</details>

File: README.md [hunk 2]
```diff
@@ -32,6 +32,40 @@ For the moment, to build this project you should
...
+* Select *Formatter*
+* Select the *Import...* button
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/eclipse/OpenHospital-Java-CodeStyle-Formatter.xml`
+* Select *Open*
+* At this point the code style is stored and is applicable to all projects opened in the IDE.  To restrict the settings just to this project select *Configure Project Specific Settings...* in the upper right.  In the next dialog select the *openhospital* repository and select *OK*.  In the next dialog select the *Enable project specific settings* checkbox.  Finally select *Apply and Close*.
+* Back in the *Code Style* menu area, select *Organize Imports*
+* Select *Import...*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/eclipse/OpenHospital.importorder`
+* Select *Open*
+* As with the formatting styles the import order is applicable to all projects.  In order to change it just for this project repeat the same steps as above for *Configure Project Specific Settings...*
```
Comment (mwithi) [inline README.md:67]: I would suggest a collapsable section for the list, like this:

<details>
  <summary>Eclipse instructions</summary>

* list1
* list2

</details>

**Review by mwithi — APPROVED** [review]
LGTM, If you like you can apply suggested changes (see comments)

**Comment by mwithi** [comment]
@pviotti should we introduce GitHub Action also here and remove Travic CI?

**Comment by mwithi** [comment]
> The failure is caused by a dependency issue as `OH-core:jar:1.9.jar` is not found:
> 
> `[ERROR] Failed to execute goal on project openhospital-api: Could not resolve dependencies for project org.isf:openhospital-api:jar:0.0.1: Could not find artifact org.isf:OH-core:jar:1.9 in google-maven-central (https://maven-central.storage-download.googleapis.com/maven2/) -> [Help 1]`

Sorry, I read later, we know there are some issues with Travic CI, so we replaced it with GitHub Actions in other repos

File: README.md (hunk 2 above)
Comment (dbmalkovsky) [reply README.md:73]: Done for both IDEs

**Pull request merged**